nixos/modules/virtualisation/digital-ocean-config.nix

```
@@ -41,7 +41,7 @@ with lib;
         kernelParams = [ "console=ttyS0" "panic=1" "boot.panic_on_fail" ];
         initrd.kernelModules = [ "virtio_scsi" ];
         kernelModules = [ "virtio_pci" "virtio_net" ];
-        loader.grub.devices = lib.mkDefault ["/dev/vda"];
+        loader.grub.devices = ["/dev/vda"];
```

To fix the error that you see, we only need this:

```suggestion
        loader.grub.devices = ["/dev/vda"];
```

---

The issue is this line: https://github.com/NixOS/nixpkgs/blob/e73004a744e8e8244d2e31c10931aad9610ec5d7/nixos/modules/system/boot/loader/grub/grub.nix#L716

---

Okay updated the PR to only update the one affected line.

> The issue is this line overriding our current default: 

Do you think it makes sense to reverse the mkDefault change instead of cleanly fixing it in grub.nix?

---

Longterm probably but let's fix the regression first. The other fix also requires deprecation of the device option.